style.css

```diff
@@ -0,0 +1,188 @@
+body { font-family: "3rd party font", serif; }
+
+.Mobile_background {
+  width: 365px;
+  height: 5410px;
+  left: 0;
+  background: #f7f7f9;
+}
+
+header {
+  position: static;
+  width: 375px;
+  height: 92px;
+  left: 0;
+  top: 0;
```

Comment:
- [ ] [Optional] I strongly recommend you to not use any `left`, `top`, `right` or `bottom` property unless is really necessary. This will make hard for you to create a responsive design